utils/underscore.js

```diff
@@ -0,0 +1,33 @@
+const _ = module.exports = require('underscore');
+
+_.mixin({
+  /**
+   * Get a deeply nested object property.
+   *
+   * @param {Object} obj The object.
+   * @param {String} path The path within the object to fetch.
+   * @param {*} fallbackValue The value to return if given path not found.
+   *
+   * @return {*} Returns value if found; otherwise the fallbackVAlue.
+   */
+  get: function(obj, path, fallbackValue) {
```

Comment:
Like you said, it may break some other stuff (e.g. I had to alias a couple of lodash methods to ensure minimongo kept working). I think at some point we should definitely switch to lodash though I'm not yet familiar with enough of the codebase to say with certainty that it won't break something. Perhaps you have a better idea of this.